packages/cozy-harvest-lib/src/models/ConnectionFlow.js

```diff
@@ -184,6 +185,7 @@ export class ConnectionFlow {
   }
 
   getKonnectorSlug() {
+    // TODO: Use this.konnector.slug?
```

Comment:
is it still needed?

Reply:
No, sometimes connectionFlow can be initialized without any konnector